runway/data_types.py

```diff
@@ -70,13 +101,17 @@ def setup(opts):
     :param name: The name associated with this variable, defaults to "{item_type}_array" \
         (e.g. "image_array")
     :type name: string, optional
+    :param description: A description of this variable and how its used in the model,
+        defaults to None
+    :type description: string, optional
     :param min_length: The minimum number of elements required to be in the array, defaults to 0
     :type min_length: int, optional
     :param max_length: The maximum number of elements allowed to be in the array, defaults to None
     :type max_length: int, optional
     :raises MissingArgumentError: A missing argument error if item_type is not specified
     """
-    def __init__(self, item_type=None, name=None, min_length=0, max_length=None):
+    def __init__(self, item_type=None, name=None, description=None, min_length=0, max_length=None):
+        super(array, self).__init__('array', name=name, description=description)
```

Comment:
To remove the keyword argument repetition of name & description we can capture and pass along `**kwargs` to the base type.